test/Microsoft.ComponentDetection.Detectors.Tests/MavenParsingUtilitiesTests.cs

```diff
@@ -79,10 +79,16 @@ public void GenerateDetectedComponentAndIsDeveDependencyAndDependencyScope_Devel
     }
 
     [TestMethod]
-    public void GenerateDetectedComponentAndIsDeveDependencyAndDependencyScope_InvalidScope()
+    public void GenerateDetectedComponentAndIsDeveDependencyAndDependencyScope_InvalidScopeIsEvaluatedAsCompile()
```

Comment:
nit: typo in test name `DeveDependency`

Reply:
fixed the typo for this test and all the other tests in the class.